packages/jsx-email/src/components/qrcode.tsx

```diff
@@ -0,0 +1,47 @@
+import * as React from 'react';
+
+const qrcode = require('qrcode');
```

Comment:
```suggestion
import qrcode from 'qrcode';
```

We'll want to avoid `require` because it won't play nice with `vite` (which is what our preview app runs on)